cmd/serve/main.go

```diff
@@ -45,8 +45,8 @@ func Register(app *kingpin.Application) *servecmd {
 	cli.Flag("external-host", "The external origin of the server (e.g. https://mydomain.com)").Envar("WG_EXTERNAL_HOST").StringVar(&cmd.AppConfig.ExternalHost)
 	cli.Flag("storage", "The storage backend connection string").Envar("WG_STORAGE").Default("memory://").StringVar(&cmd.AppConfig.Storage)
 	cli.Flag("disable-metadata", "Disable metadata collection (i.e. metrics)").Envar("WG_DISABLE_METADATA").Default("false").BoolVar(&cmd.AppConfig.DisableMetadata)
-	cli.Flag("disable-inactive", "Disable inactive device deletion").Envar("WG_DISABLE_INACTIVE").Default("false").BoolVar(&cmd.AppConfig.DisableInactive)
-	cli.Flag("inactive-duration", "Duration after inactive device are deleted").Envar("WG_INACTIVE_DURATION").Default("4380h").DurationVar(&cmd.AppConfig.InactiveDuration)
+	cli.Flag("enable-inactive-device-deletion", "Enable inactive device deletion").Envar("WG_ENABLE_INACTIVE_DEVICE_DELETION").Default("false").BoolVar(&cmd.AppConfig.EnableInactiveDeviceDeletion)
+	cli.Flag("inactive-device-grace-period", "Duration after inactive device are deleted").Envar("WG_INACTIVE_DEVICE_GRACE_PERIOD").Default("4380h").DurationVar(&cmd.AppConfig.InactiveDeviceGracePeriod)
```

Comment:
The `"4380h"` are kind of hard to parse. I am aware that there is no day (`d`) with these durations, but could we introduce a variable for it, e.g.:
```
var HALF_YEAR = "4380h"  
```
or `SIX_MONTHS` or something?

Reply:
We can do the same as the `time` package:

```go
const (
	Day  time.Duration = 24 * time.Hour
	Year                        = 365 * Day
)
```

and set the default to 1 year, because 6 month is kinda hard to represent and the default was chosen arbitrarily anyway.